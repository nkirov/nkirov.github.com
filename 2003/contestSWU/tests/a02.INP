20
A01
B02
C03
D04
E05
F06
G07
H08
I09
J10
A011
B021
C031
D041
E051
F061
G071
H081
I091
J101
7 11 3 12 17 18 16 19 9 8 5 20 13 1 14 2 15 4 6 10
11 3 12 17 18 16 19 9 8 5 20 13 1 14 2 15 4 6 10 7
3 12 17 18 16 19 9 8 5 20 13 1 14 2 15 4 6 10 7 11
12 17 18 16 19 9 8 5 20 13 1 14 2 15 4 6 10 7 11 3
17 18 16 19 9 8 5 20 13 1 14 2 15 4 6 10 7 11 3 12
18 16 19 9 8 5 20 13 1 14 2 15 4 6 10 7 11 3 12 17
16 19 9 8 5 20 13 1 14 2 15 4 6 10 7 11 3 12 17 18
19 9 8 5 20 13 1 14 2 15 4 6 10 7 11 3 12 17 18 16
9 8 5 20 13 1 14 2 15 4 6 10 7 11 3 12 17 18 16 19
8 5 20 13 1 14 2 15 4 6 10 7 11 3 12 17 18 16 19 9
5 20 13 1 14 2 15 4 6 10 7 11 3 12 17 18 16 19 9 8
20 13 1 14 2 15 4 6 10 7 11 3 12 17 18 16 19 9 8 5
13 1 14 2 15 4 6 10 7 11 3 12 17 18 16 19 9 8 5 20
1 14 2 15 4 6 10 7 11 3 12 17 18 16 19 9 8 5 20 13
14 2 15 4 6 10 7 11 3 12 17 18 16 19 9 8 5 20 13 1
2 15 4 6 10 7 11 3 12 17 18 16 19 9 8 5 20 13 1 14
15 4 6 10 7 11 3 12 17 18 16 19 9 8 5 20 13 1 14 2
4 6 10 7 11 3 12 17 18 16 19 9 8 5 20 13 1 14 2 15
6 10 7 11 3 12 17 18 16 19 9 8 5 20 13 1 14 2 15 4
10 7 11 3 12 17 18 16 19 9 8 5 20 13 1 14 2 15 4 6
