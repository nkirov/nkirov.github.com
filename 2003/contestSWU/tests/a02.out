A01
B02
C03
D04
E05
F06
G07
H08
I09
J10
A011
B021
C031
D041
E051
F061
G071
H081
I091
J101
